src/mel.jl

```diff
@@ -65,39 +65,43 @@ function mel_frequencies(nmels::Int = 128, fmin::Real = 0.0f0, fmax::Real = 1102
     min_mel = hz_to_mel(fmin)[1]
     max_mel = hz_to_mel(fmax)[1]
 
-    mels = linspace(min_mel, max_mel, nmels)
+    mels = range(min_mel, max_mel, length=nmels)
     mel_to_hz(mels)
 end
 
 """"""
 function mel(samplerate::Real, nfft::Int, nmels::Int = 128, fmin::Real = 0f0, fmax::Real = samplerate/2f0)
-    weights = zeros(Float32, nmels, (nfft >> 1) + 1)
+    second_dim = ((nfft >> 1) + 1)
+    weights = zeros(Float32, nmels, second_dim)
     fftfreqs = fft_frequencies(samplerate, nfft)
     melfreqs = mel_frequencies(nmels + 2, fmin, fmax)
-    enorm = 2f0 ./ (melfreqs[3:end] - melfreqs[1:nmels])
+    enorm = 2f0 ./ (melfreqs[3:end] .- melfreqs[1:nmels])
 
     for i in 1:nmels
-        lower = (fftfreqs - melfreqs[i]) / (melfreqs[i+1] - melfreqs[i])
-        upper = (melfreqs[i+2] - fftfreqs) / (melfreqs[i+2] - melfreqs[i+1])
+        lower = (fftfreqs .- melfreqs[i]) ./ (melfreqs[i+1] .- melfreqs[i])
+        upper = (melfreqs[i+2] .- fftfreqs) ./ (melfreqs[i+2] .- melfreqs[i+1])
```

Comment:
```suggestion
        lower = (fftfreqs .- melfreqs[i]) ./ (melfreqs[i+1] - melfreqs[i])
        upper = (melfreqs[i+2] .- fftfreqs) ./ (melfreqs[i+2] - melfreqs[i+1])
```
this is scalar minus I think

Reply:
it doesn't make any difference in the actual code though